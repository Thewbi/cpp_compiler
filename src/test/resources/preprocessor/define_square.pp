#define SQUARE(x) ((x) * (x))


//#if defined(_DEBUG) || defined(_UNIT_TEST)

int main() {
  printf("Square of 4: %d\n", SQUARE(4));
  return 0;
}
/**/
