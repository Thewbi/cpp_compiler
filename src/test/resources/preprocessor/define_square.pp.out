intmain(){  printf("Square of 4: %d\n",SQUARE(4));  return0;}